app/frontend/src/components/admin/AdminFormsTable.vue

```diff
@@ -1,24 +1,29 @@
 <template>
-  <div>
+  <div :class="{ 'dir-rtl': isRTL }">
     <v-row no-gutters>
-      <v-col cols="12" sm="8">
+      <v-col cols="6">
         <v-checkbox
           class="pl-3"
+          :class="isRTL ? 'float-right' : 'float-left'"
           v-model="activeOnly"
           :label="$t('trans.adminFormsTable.showDeletedForms')"
           @click="refeshForms"
         />
       </v-col>
-      <v-col cols="12" sm="4">
+      <v-col cols="6">
         <!-- search input -->
-        <div class="submissions-search">
+        <div
+          class="submissions-search"
+          :class="isRTL ? 'float-left' : 'float-right'"
+        >
           <v-text-field
             v-model="search"
             append-icon="mdi-magnify"
             :label="$t('trans.adminFormsTable.search')"
             single-line
             hide-details
             class="pb-5"
+            :class="[{ 'dir-rtl': isRTL }, isRTL ? 'label' : null]"
```

Comment:
I'm just curious, is:

> [{ 'dir-rtl': isRTL }, isRTL ? 'label' : null]

the same as:

> { 'dir-rtl': isRTL, 'label': isRTL }

Reply:
This should work. I did not think that way. Thanks

Reply:
What would you think about replacing all the ternary operators that have "null" as the second argument with the "object" style? I think it's a bit easier to read, and more consistent?